src/software/ai/navigator/path_planner/path_planner_test.cpp

```diff
@@ -35,94 +35,85 @@ RobotNavigationObstacleFactory robot_navigation_obstacle_factory(
-                      {std::cos(3 * M_PI / 3) * 0.2, std::sin(3 * M_PI / 3) * 0.2})),
-                  robot_navigation_obstacle_factory.createFromRobotPosition(Point(
-                      {std::cos(4 * M_PI / 3) * 0.2, std::sin(4 * M_PI / 3) * 0.2})),
-                  robot_navigation_obstacle_factory.createFromRobotPosition(Point(
-                      {std::cos(5 * M_PI / 3) * 0.2, std::sin(5 * M_PI / 3) * 0.2})),
-              },
-          .should_return_path = false},
-         {.name  = "Start inside a rectangular obstacle, end is outside of obstacle",
-          .start = Point(0, 0),
-          .end   = Point(4, 0),
-          .navigable_area     = Rectangle({-5, -5}, {5, 5}),
-          .obstacles          = {robot_navigation_obstacle_factory.createFromShape(
-              Rectangle({-1, -1}, {1, 1}))},
-          .should_return_path = true},
-         {.name = "Start and end inside same obstacle",
-          // NOTE: this test is designed specifically to pass the progress check
-          .start              = Point(0, 0),
-          .end                = Point(1.5, 0),
-          .navigable_area     = Rectangle({-5, -5}, {5, 5}),
-          .obstacles          = {robot_navigation_obstacle_factory.createFromShape(
-              Rectangle({-1, -1}, {2, 1}))},
-          .should_return_path = true}};
+                  {.name           = "Path blocked by Rectangle",
+                   .start          = Point(-4, 0),
+                   .end            = Point(4, 0),
+                   .navigable_area = Rectangle({-5, -5}, {5, 5}),
+                   .obstacles      = {robot_navigation_obstacle_factory.createFromShape(
+                       Rectangle({-1, 5}, {2, -5}))},
+                   .should_return_path = false},
+
+                  {.name =
+                       "Start inside a rectangular obstacle, end is outside of obstacle",
+                   .start          = Point(0, 0),
+                   .end            = Point(4, 0),
+                   .navigable_area = Rectangle({-5, -5}, {5, 5}),
+                   .obstacles      = {robot_navigation_obstacle_factory.createFromShape(
+                       Rectangle({-1, -1}, {1, 1}))},
+                   .should_return_path = true},
+                  {.name = "Start and end inside same obstacle",
+                   // NOTE: this test is designed specifically to pass the progress check
+                   .start          = Point(0, 0),
+                   .end            = Point(1.5, 0),
+                   .navigable_area = Rectangle({-5, -5}, {5, 5}),
+                   .obstacles      = {robot_navigation_obstacle_factory.createFromShape(
+                       Rectangle({-1, -1}, {2, 1}))},
+                   .should_return_path = true}};
 
 
 template <typename PlannerT>
-std::pair<std::string, PathPlannerConstructor> nameAndConstructor()
+std::pair<std::string, PathPlannerConstructor> name_and_constructor()
```

Comment:
Why the name change here? This doesn't seem in-line with our naming conventions?

Reply:
whoops bad merge